site/content/docs/testapps/altoroj.md

```diff
@@ -12,9 +12,43 @@ maintained by [HCL Software](https://www.hcl-software.com/).
 
 It is a traditional app created in 2008 and is not updated very often.
 
-* Online: http://testfire.net/
+* Online: https://demo.testfire.net/
 * Repo: https://github.com/HCL-TECH-SOFTWARE/AltoroJ
 
+### Quick Start
+
+New to ZAP and just want to quickly run ZAP against Testfire (AltoroJ)?
+
+Just run these commands:
+
+```bash
+# Download the recommended plan using curl, or use any other suitable tool
+curl -O https://raw.githubusercontent.com/zaproxy/community-scripts/refs/heads/main/other/af-plans/FullScanTestfireAuth.yaml
+
+# Run ZAP using the stable Docker image, mapping the CWD so that Docker can access the file system and export the report
+docker run -v $(pwd):/zap/wrk/:rw -t zaproxy/zap-stable zap.sh -cmd -autorun wrk/FullScanTestfireAuth.yaml
+```
+
+To run this command on Windows see the [relevant documentation](/docs/docker/about/#mounting-the-current-directory).
+
+You will need to have Docker installed. If you do not want to use Docker then you can of course install ZAP locally.
+
+The scan should find the following High and Medium risk alerts:
+
+* 🔴 [Cross Site Scripting (Reflected](/docs/alerts/40012/)
+* 🔴 [SQL Injection](/docs/alerts/40018/)
+* 🔴 [External Redirect](/docs/alerts/20019/)
+* 🔴 [PII Disclosure](/docs/alerts/10062/)
+* 🟠 [Absence of Anti-CSRF Tokens](/docs/alerts/10202/)
+* 🟠 [Anti-CSRF Tokens Check](/docs/alerts/20012/)
+* 🟠 [Buffer Overflow](/docs/alerts/30001/) / [Integer Overflow Error](/docs/alerts/30003/)
+* 🟠 [Missing Anti-clickjacking Header](/docs/alerts/10020/)
+* 🟠 [Relative Path Confusion](/docs/alerts/10051/)
+* 🟠 [Sub Resource Integrity Attribute Missing](/docs/alerts/90003/)
+* 🟠 [Content Security Policy (CSP) Header Not Set](/docs/alerts/10038/)
```

Comment:
I'm not seeing this results with the plan provided. Was that side effect of changing the domain?

<img width="1179" height="303" alt="results" src="https://github.com/user-attachments/assets/4e67f175-d017-4e8d-8e1b-451aaec63b98" />

Reply:
I'll have to re-run and check.

Reply:
Did you use the Docker command? The example ones are unexpected.

Reply:
Ack, I got a similar reduced set with docker. Will adjust:
<img width="1140" height="366" alt="image" src="https://github.com/user-attachments/assets/2368895f-11cc-4940-bc47-6d04baf58c20" />